src/Features/CSharp/Portable/Completion/CompletionProviders/OperatorsAndIndexer/UnnamedSymbolCompletionProvider_Operators.cs

```diff
@@ -0,0 +1,177 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Completion;
+using Microsoft.CodeAnalysis.Completion.Providers;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.Completion.Providers
+{
+    internal partial class UnnamedSymbolCompletionProvider
+    {
+        [Flags]
+        private enum OperatorPosition
+        {
+            None = 0,
+            Prefix = 1,
+            Infix = 2,
+            Postfix = 4,
+        }
+
+        // Place operators after conversions.
+        private readonly int OperatorSortingGroupIndex = 2;
+
+        private readonly string OperatorName = nameof(OperatorName);
+        private readonly ImmutableDictionary<string, string> OperatorProperties =
+            ImmutableDictionary<string, string>.Empty.Add(KindName, OperatorKindName);
+
+        /// <summary>
+        /// Ordered in the order we want to display operators in the completion list.
+        /// </summary>
+        private static readonly ImmutableArray<(string name, OperatorPosition position)> s_operatorInfo1 =
```

Comment:
nit: Why it has a suffix `1`?

Reply:
ah.  forgot to change this.  this is something i often do when making an itneresting change to an existing variable.  i change the name as well so that all consumers break and i can ensure i've checked all of them for correctness (or in this case, deleted ones that became unnecessary).  I forgot to change this back.  Fixed now.